notebooks/official/tensorboard/tensorboard_custom_training_with_prebuilt_container.ipynb

```diff
@@ -0,0 +1,1004 @@
+{
```

Comment:
<p>A TensorBoard instance is created or retrieved from previous step so is there a need to run this step?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/603/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
<p>OK I will remove it</p>